ocaml/testsuite/tests/typing-implicit-source-positions/let_operators.ml

```diff
@@ -0,0 +1,62 @@
+(* TEST
```

Comment:
This is a good idea embodied in this file. It would be cool if all of the examples here (both with source positions and with ordinary optionals) worked. Do you think we'd use this feature within Jane Street? I don't think it would be hard to make work, but not quite trivial either.

Reply:
> Do you think we'd use this feature within Jane Street?

Ooh good question! I think it would not be used within Jane Street as `ppx_js_style` forbids using let operators in favor of `ppx_let`.

Reply:
OK. Let's not do this, then. Maybe when upstreaming, we can take a stab. Instead, leave comments (plain ones -- no `CR`) saying that the examples would be good to accept, but we're not going to for now.